### PR TITLE
#412 useDispatch

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**
publish useDispatch which is a shortcut for const {dispatch} = useContext<TaipyStore>(TaipyContext)

### CODE REVIEW

**Review by florian-vuillemot — APPROVED** [review]
LGTM

File: gui/packaging/taipy-gui.d.ts [hunk 1]
```diff
@@ -11,6 +11,8 @@
  * specific language governing permissions and limitations under the License.
  */
 
+import * as React from "react";
```
Comment (FabienLelaquais) [inline gui/packaging/taipy-gui.d.ts:14]: I don't believe you wrote that

**Review by FabienLelaquais — APPROVED** [review]
Good move

**Review by dinhlongviolin1 — APPROVED** [review]

File: gui/packaging/taipy-gui.d.ts (hunk 1 above)
Comment (FredLL-Avaiga) [reply gui/packaging/taipy-gui.d.ts:14]: I did

**Pull request merged**